data/gnudata.g

```diff
@@ -51,6 +51,7 @@ GNU_SAVE( 3402, 1568, WITH_GC );
 GNU_SAVE( 3600, 4484, WITH_GC );
 GNU_SAVE( 3888, 20065, WITH_GC );
 GNU_SAVE( 4752, 2919, WITH_GC );
+GNU_SAVE( 8353, 5847, WITH_GC );
```

Comment:
A duty of the reviewer is not only to check the value of gnu(n) but also check that the line setting the data is correct. In this case it has a typo.

I recommend now to use the function `GnuByConstructAllGroups` from `lib/grpconst.g`. It prints the text to copy and paste into submission. For example, here it tells:
```
****************************************
gnu(8352)=5847

GAP      4.8.3
GrpConst 2.5
Runtime: 3626067 ms
Isomorphic groups eliminated

In case this value is new, add the next line to data/gnudata.g
GNU_SAVE( 8352, 5847, WITH_GC );

****************************************
```
This makes submitting new values faster and reduces the chance of a typo. This function also returns the constructed list, so you may want to call it with double semicolon to avoid many screens of output.